File: fm-domain/src/main/java/io/zeitwert/fm/building/adapter/api/rest/dto/TransferMetaDto.java
package io.zeitwert.fm.building.adapter.api.rest.dto;

import lombok.Builder;
import lombok.Data;

import java.time.OffsetDateTime;

@Data()
@Builder
public class TransferMetaDto {

	private String aggregate;
	private String version;

	private String createdByUser;
	private OffsetDateTime createdAt;

	private String modifiedByUser;
	private OffsetDateTime modifiedAt;

}

File: fm-domain/src/main/java/io/zeitwert/fm/building/adapter/api/rest/dto/TransferMetaDto.kt
package io.zeitwert.fm.building.adapter.api.rest.dto

import java.time.OffsetDateTime

data class TransferMetaDto(
    var aggregate: String? = null,
    var version: String? = null,
    var createdByUser: String? = null,
    var createdAt: OffsetDateTime? = null,
    var modifiedByUser: String? = null,
    var modifiedAt: OffsetDateTime? = null,
)

